### PR TITLE
Add searchable string for ServiceOffering operations logs 

### PR DESCRIPTION
**Pull request by lpichler**
for order operation is the string:
`ServiceOffering#order`

cc @slemrmartin

### CODE REVIEW

File: app/controllers/api/v1/service_offerings_controller.rb [hunk 1]
```diff
@@ -19,34 +19,34 @@ def applied_inventories
...
+        logger.info("ServiceOffering##{operation_type}: ServiceOffering(id: #{service_offering.id}, source_ref: #{service_offering.source_ref}), Task(id: #{task.id}): event(ServiceOffering.#{operation_type}) published to kafka.")
 
         render :json => {:task_id => task.id}
       rescue ActiveRecord::RecordNotFound => err
-        logger.error("#{err.message}")
+        logger.error("ServiceOffering##{operation_type}: #{err.message}")
         head :bad_request
       rescue StandardError => err
-        logger.error("#{task ? "Task(id: #{task.id}):" : ""} #{err.message} (#{err.class})")
+        logger.error("ServiceOffering##{operation_type}: ServiceOffering(id: #{operation_params[:service_offering_id].to_i }, source_ref: #{service_offering&.source_ref}), #{task ? "Task(id: #{task.id}):" : ""}#{err.message} (#{err.class})")
```
Comment (slemrmartin) [inline app/controllers/api/v1/service_offerings_controller.rb]: task could be nil there

Comment (lpichler) [reply app/controllers/api/v1/service_offerings_controller.rb]: I think I count with it in expression `#{task ? "Task(id: #{task.id}):" : ""}` or did I miss other place ?

Comment (slemrmartin) [reply app/controllers/api/v1/service_offerings_controller.rb]: ah, I lost in in line-break, sorry :)

**Comment by syncrou** [comment]
@slemrmartin @lpichler was there a Jira ticket tied to this change?

**Comment by lpichler** [comment]
@syncrou I am using this TPINVTRY-838 but I have more PRs related to this task,
Should I have to add them to the Jira Ticket ?

**Pull request merged**